index.js

```diff
@@ -62,10 +62,11 @@ function morgan (format, options) {
 
   if (format && typeof format === 'object') {
     opts = format
+
     fmt = opts.format || 'default'
 
     // smart deprecation message
-    deprecate('morgan(options): use morgan(' + (typeof fmt === 'string' ? JSON.stringify(fmt) : 'format') + ', options) instead')
+    deprecate('morgan(options): use morgan(' + (typeof opts.format === 'string' ? JSON.stringify(opts.format) : 'format') + ', options) instead')
```

Comment:
Nice! This makes a lot of sense -- it's basically just telling you to use `(format, options)` instead of echoing a format you didn't provide. That makes a lot of sense.